### PR TITLE
feat: global override for all operations in check

### PR DESCRIPTION
**Pull request by thisisnithin**
## Motivation and Context

https://github.com/user-attachments/assets/55325736-815d-4714-8c44-4c980ed2e19f

## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: controlplane/src/core/bufservices/PlatformService.ts [hunk 1]
```diff
@@ -3256,6 +3257,175 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
...
+        if (!graph) {
+          return {
+            response: {
+              code: EnumStatusCode.ERR_NOT_FOUND,
+              details: 'Requested graph does not exist',
+            },
+          };
+        }
+
+        return await opts.db.transaction(async (tx) => {
```
[the remark this answers is not in the document]
Comment (thisisnithin) [reply controlplane/src/core/bufservices/PlatformService.ts:3362]: I do

[the remark this answers is not in the document]
Comment (thisisnithin) [reply controlplane/src/core/bufservices/PlatformService.ts:3362]: Done

File: controlplane/src/core/bufservices/PlatformService.ts [hunk 2]
```diff
@@ -3256,6 +3257,175 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
       });
     },
 
+    createIgnoreOverridesForAllOperations: (req, ctx) => {
```
[the remark this answers is not in the document]
Comment (thisisnithin) [reply controlplane/src/core/bufservices/PlatformService.ts:3260]: It is very clear this way

File: controlplane/src/core/bufservices/PlatformService.ts [hunk 3]
```diff
@@ -3256,6 +3257,175 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
...
+          return {
+            response: {
+              code: EnumStatusCode.OK,
+            },
+          };
+        });
+      });
+    },
+
+    toggleChangeOverridesForAllOperations: (req, ctx) => {
```
[the remark this answers is not in the document]
Comment (thisisnithin) [reply controlplane/src/core/bufservices/PlatformService.ts:3333]: Those should be separate things. A method is doing too much by combining them.

File: controlplane/src/core/bufservices/PlatformService.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (StarpTech) [reply controlplane/src/core/bufservices/PlatformService.ts:3260]: ok

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**